tasks/bootstrap.yml

```diff
@@ -22,9 +22,19 @@
   ignore_errors: "{{ ansible_check_mode }}"
   tags: configure
 
+- name: Check for wsrep in systemctl MYSQLD_OPTS environment option
+  shell: 'systemctl show-environment|grep -q -e "^MYSQLD_OPTS=.*--wsrep_on=OFF.*"'
```

Comment:
You should use '-E' instead of '-e'

Reply:
or egrep

Reply:
It works like this but if you prefer, I change it.